packages/core/src/Checkbox/CheckboxRoot.vue

```diff
@@ -25,7 +25,7 @@ export interface CheckboxRootProps extends PrimitiveProps, FormFieldProps {
 
 export type CheckboxRootEmits = {
   /** Event handler called when the value of the checkbox changes. */
-  'update:modelValue': [value: AcceptableValue ]
+  'update:modelValue': [value: boolean | 'indeterminate']
```

Comment:
Checkbox now accept other types, not only `boolean` in v2.

```suggestion
  'update:modelValue': [value: AcceptableValue | 'indeterminate']
```

Reply:
TS in my project is failing because AcceptableValue doesn't include `boolean`. I have copied the types of the prop : https://github.com/unovue/radix-vue/blob/76923e561b89e648ca7218f1195ff92031996d30/packages/core/src/Checkbox/CheckboxRoot.vue#L14
but I may be missing something

Reply:
There is the boolean casting quirk commented in AcceptableValue : https://github.com/unovue/radix-vue/blob/fc3927037b591a4869ea3b2414bdaff1af2f6d9f/packages/core/src/shared/types.ts#L51-L53

Reply:
Ahh you are right. Then let's make it ` [value: AcceptableValue | boolean | 'indeterminate']`

Reply:
I have changed to  `AcceptableValue | boolean | 'indeterminate'` and added generic.